### DISCUSSION TITLE
How to pronounce 'kyuubi'?

### DISCUSSION THREAD

**Discussion by yx91490** [apache/kyuubi#445] [Q&A]
kyuubi 发音是什么？

**Comment by yaooqinn** [comment]
The project took its name from a character of a  popular Japanese manga  - Naruto. The character is named Kyuubi Kitsune/ Kurama, which is a nine-tailed fox. Both the manga and the character spread the power of persistence and **FIRE** (referring to the project - Apache Spark). The nine-tail stands for multi-tenancy support of this project.

How to pronounce: https://forvo.com/word/kyuubi_kitsune/ or https://zh.howtopronounce.com/kyuubi

![image](https://user-images.githubusercontent.com/8326978/101899446-cf38f500-3be8-11eb-98ce-f600c580425c.png)

**Comment by yx91490** [comment]
sounds like ['kjuːbi]😄

**Comment by yaooqinn** [comment]
Yea, that's it!
